src/main/java/bhs/devilbotz/commands/DriveStraight.java

```diff
@@ -29,7 +33,7 @@ public void initialize() {
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
-    drivetrain.arcadeDrive(0.1, 0);
+    drivetrain.arcadeDrive(1, 0);
```

Comment:
What does this constant do?  Does it cause the robot to drive straight at max speed?  We should see how this actually works on the physical robot.  E.g. we may want to ramp up the voltage so that it accelerates smoothly.

Reply:
Maybe add a slew rate limiter?

Reply:
I don't think this command is going to be used.  It was part of exercise of teaching why we need to use some kind of control system to drive a set distance or stay straight.  We could just delete this file? Or leave it for educational/reference purpose? Either way is fine.